File: movieshows/src/main/java/com/niranjandroid/movieshows/movielisting/MovieListingContract.java
package com.niranjandroid.movieshows.movielisting;

import com.niranjandroid.movieshows.base.BaseContract;

/**
 * Created by Niranjan P on 8/20/2017.
 */

public interface MovieListingContract {
    interface View extends BaseContract.View{

    }

    interface Presenter extends BaseContract.Presenter {
        void attachView(MovieListingContract.View view);
    }

    interface Interactor {
    }
}

File: movieshows/src/main/java/com/niranjandroid/movieshows/movielisting/MovieListingContract.kt
package com.niranjandroid.movieshows.movielisting

import com.niranjandroid.movieshows.base.BaseContract

/**
 * Created by Niranjan P on 8/20/2017.
 */

interface MovieListingContract {
    interface View : BaseContract.View

    interface Presenter : BaseContract.Presenter {
        fun attachView(view: MovieListingContract.View)
    }

    interface Interactor
}
